app/views/state_file/questions/ny_primary_state_id/_state_id.html.erb

```diff
@@ -56,6 +56,7 @@
       </div>
     </div>
   </div>
+  <%= content_for :info_box %>
```

Comment:
Not ideal but a concession I made to not interfere with the reusable templates more than I have to.